README.md

```diff
@@ -8,7 +8,70 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running a demo on Windows
```

Comment:
```suggestion
## Running the demo on Windows system
```

Reply:
Changed it to "Running the demo on Windows"